robocop/checkers/spacing.py

```diff
@@ -0,0 +1,119 @@
+"""
+Spacing checkers
+"""
+from robot.parsing.model.blocks import TestCase, Keyword
+from robot.parsing.model.statements import EmptyLine, Comment
+from robocop.checkers import RawFileChecker, VisitorChecker
+from robocop.rules import RuleSeverity
+
+
+def register(linter):
+    linter.register_checker(InvalidSpacingChecker(linter))
+    linter.register_checker(MissingTrailingBlankLineChecker(linter))
+
+
+class InvalidSpacingChecker(RawFileChecker):
+    """ Checker for invalid spacing. """
+    rules = {
+        "1001": (
+            "trailing-whitespace",
+            "Trailing whitespace at the end of line",
+            RuleSeverity.WARNING
+        )
+    }
+
+    def check_line(self, line, lineno):
```

Comment:
We can reuse that checker for perfomance and to implement missing-trailing-blank-line or mixed spaces/tabs rules. 

For example given this:
```
    def __init__(self, *args):
        self.lines = []
        super().__init__(*args)

   def visit_File(self, node):
       self.lines = []
       super().visit_File(node)
       # check self.lines[-1] for missign trailing line, or too much of it

   def check_line(self, line, lineno):
       # do stuff
       self.lines.append(line)

Reply:
I thought about storing lines in a list but didn't have time yesterday to do it. Good idea 😃